py/query.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+
+'''
+If you're running this from this directory you can start the server with the following command:
+./query.py localhost:8005
+
+sample url looks like this:
+http://localhost:8005/query_by_ids?json={"segments":[345678,345780,345795]}'''
```

Comment:
KVP will be simpler for clients to produce. (I'm thinking ahead to when we hook this up to a front-end UI.) But I'll defer to you two.